x-pack/plugins/apm/public/components/app/correlations/context_popover/top_values.tsx

```diff
@@ -67,103 +231,65 @@ export function TopValues({ topValueStats, onAddFilter, fieldValue }: Props) {
-                      }}
-                    />
-                    <EuiButtonIcon
-                      iconSize="s"
-                      iconType="minusInCircle"
-                      onClick={() => {
-                        onAddFilter({
-                          fieldName,
-                          fieldValue:
-                            typeof value.key === 'number'
-                              ? value.key.toString()
-                              : value.key,
-                          include: false,
-                        });
-                      }}
-                      aria-label={i18n.translate(
-                        'xpack.apm.correlations.fieldContextPopover.removeFilterAriaLabel',
-                        {
-                          defaultMessage: 'Filter out {fieldName}: "{value}"',
-                          values: { fieldName, value: value.key },
-                        }
-                      )}
-                      data-test-subj={`apmFieldContextTopValuesExcludeFilterButton-${value.key}-${value.key}`}
-                      style={{
-                        minHeight: 'auto',
-                        width: theme.eui.euiSizeL,
-                        paddingTop: 0,
-                        paddingBottom: 0,
-                        paddingRight: 2,
-                        paddingLeft: 2,
-                      }}
-                    />
-                  </>
-                ) : null}
-              </EuiFlexGroup>
+              <TopValue
+                value={value}
+                barColor={barColor}
+                valueText={valueText}
+                onAddFilter={onAddFilter}
+                progressBarMax={progressBarMax}
+                isHighlighted={isHighlighted}
+                fieldName={fieldName}
+              />
             </>
           );
         })}
+
+      {Array.isArray(fieldValueStats?.topValues) && (
+        <>
+          <EuiHorizontalRule margin="s" />
+          <EuiSpacer size="s" />
+          <EuiText size="xs">
+            <FormattedMessage
+              id="xpack.apm.correlations.fieldContextPopover.notTopTenValueMessage"
+              defaultMessage="Selected term is not in the top 10"
+            />
+          </EuiText>
+
+          {fieldValueStats?.topValues.map((value) => {
+            const valueText =
+              progressBarMax !== undefined
+                ? asPercent(value.doc_count, progressBarMax)
+                : undefined;
+
+            return (
+              <TopValue
+                value={value}
+                barColor={'accent'}
+                valueText={valueText}
+                onAddFilter={onAddFilter}
+                progressBarMax={progressBarMax}
+                isHighlighted={true}
+                fieldName={fieldName}
+              />
+            );
+          })}
+        </>
+      )}
+
+      {topValueStats.topValuesSampleSize !== undefined && (
+        <Fragment>
```

Comment:
You can just use `<>`. You only need to import `Fragment` if you'll need to give it a `key`.

Reply:
Updated here https://github.com/elastic/kibana/pull/118069/commits/58d8bcc2f17926342d614fbf4049f2d2cfcc0749